modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -79,25 +79,25 @@ describe('browser.tinymce.core.EditorTest', () => {
     editor.options.set('relative_urls', true);
     editor.documentBaseURI = new URI('http://www.site.com/dirA/dirB/dirC/');
 
-    editor.setContent('<a href="test.html">test</a>');
+    editor.setContent('<p><a href="test.html">test</a></p>');
```

Comment:
Are we sure we want this? I honestly would have expected this to be unnecessary and it to keep it as an inline anchor in this case since it doesn't contain another block element, but I guess that might be difficult to know what the intention was.

I'm also concerned this could be a **breaking** change if someone has this content and loads it into the editor is all. I guess it shouldn't happen when loading in existing content though, only new content.

Reply:
A lot of these changes is to avoid having tests that rely on forced root blocks when they are testing something else. That is why I changed a lot of them. So it doesn't set one thin then tests for another thing.